.gitignore

```diff
@@ -0,0 +1,2 @@
+# IDE files

```

Comment:
hm, I'm not sure a .gitignore is necessary for these files - it's easy enough to pick them out of a PR.